git/repo.go

```diff
@@ -86,7 +86,7 @@ func (r Repo) Clone(c Config) (*Checkout, error) {
 	}
 
 	notesRef, err := getNotesRef(repoDir, c.NotesRef)
-	if err != nil {
+	if err != nil && !strings.Contains(err.Error(), "Couldn't find remote ref") {
```

Comment:
getNotesRef uses `execGitCmd` to get the notes. It's this that is erroring out:
https://github.com/weaveworks/flux/blob/8283d84b2b8ec26f72ce5154fdd380c4b9933286/git/operations.go#L91
Yes, it definitely removed the warning.

I'm wondering whether we should do this check on the main `execGitCmd` instead. It seems like the check should apply to every git error command. "If it does error because of this, we don't care".